pages/dashboard/views/index.jsx

```diff
@@ -35,6 +35,13 @@ const Index = ({
     {
       !!profile.establishments.length && <Fragment>
         <h3>Establishments</h3>
+        {
+          !!profile.invitations.length && <Fragment>
+            <h3>Pending Invitations</h3>
+            <PanelList panels={profile.invitations.map(invitation => <Invitation key={invitation.id} establishment={ invitation.establishment.name } token={invitation.token} />)}/>
+          </Fragment>
+        }
```

Comment:
This needs to come outside of the condition about `establishments.length` - it's very probable that people will have invitations while having no establishments.

Reply:
done